app/controllers/application_controller.rb

```diff
@@ -1,18 +1,13 @@
 class ApplicationController < ActionController::Base
-  before_action :set_current_user
+  before_action :configure_permitted_parameters, if: :devise_controller?
 
```

Comment:
- [ ] It would be nice to add the `authenticate_user!` here so that users who are not logged in cannot visit our pages.

  ```
  before_action :authenticate_user!
  ```

Reply:
![image](https://github.com/zuhebahmed88091/Blog_App_Rails/assets/88501552/7900e1eb-17f2-4893-bda5-743f2fdc5e31)

Resolved